enterprise/frontend/src/embedding-sdk/test/sdk-compatibility-warning.unit.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import { render } from "@testing-library/react";
+import fetchMock from "fetch-mock";
+
+import {
+  setupCurrentUserEndpoint,
+  setupPropertiesEndpoints,
+} from "__support__/server-mocks";
+import { waitForLoaderToBeRemoved } from "__support__/ui";
+import {
+  MetabaseProvider,
+  defineEmbeddingSdkConfig,
+} from "embedding-sdk/components/public";
+import {
+  createMockSettings,
+  createMockTokenFeatures,
+  createMockUser,
+  createMockVersion,
+} from "metabase-types/api/mocks";
+
+import { getEmbeddingSdkVersion } from "../config";
+
+// TODO: extract this common setup to a shared util
+const METABASE_INSTANCE_URL = "path:";
+const AUTH_PROVIDER_URL = "http://auth-provider/metabase-sso";
+
+const defaultAuthUriConfig = defineEmbeddingSdkConfig({
+  metabaseInstanceUrl: METABASE_INSTANCE_URL,
+  authProviderUri: AUTH_PROVIDER_URL,
+  fetchRequestToken: _ =>
+    Promise.resolve({
+      id: "123",
+      exp: Number.MAX_SAFE_INTEGER,
+    }),
+});
+
+jest.mock("../config", () => ({
+  getEmbeddingSdkVersion: jest.fn(),
```

Comment:
We don't usually mock imports, but the sdk version is hardcoded at build time so I don't think we have other practical options